From e48f5483eee7f3ecaa7a242ae4e47fafc44ed216 Mon Sep 17 00:00:00 2001
From: Joerg Hofrichter <joerg.hofrichter@ni.com>
Date: Tue, 4 Jun 2024 08:44:40 +0200
Subject: [PATCH] systemd: don't set SOURCE_DATE_EPOCH to 0

This leads to the following warning when booting: "Current system time is
further ahead 15y after build time, but cannot correct: Invalid argument"

Signed-off-by: Joerg Hofrichter <joerg.hofrichter@ni.com>
---
 meta-mender-core/recipes-core/systemd/systemd_%.bbappend | 8 --------
 1 file changed, 8 deletions(-)
 delete mode 100644 meta-mender-core/recipes-core/systemd/systemd_%.bbappend

diff --git a/meta-mender-core/recipes-core/systemd/systemd_%.bbappend b/meta-mender-core/recipes-core/systemd/systemd_%.bbappend
deleted file mode 100644
index 1b83b65e..00000000
--- a/meta-mender-core/recipes-core/systemd/systemd_%.bbappend
+++ /dev/null
@@ -1,8 +0,0 @@
-# Avoid issues with time being out of sync on first boot.  By default,
-# systemd uses its build time as the epoch. When systemd is launched
-# on a system without a real time clock, this time will be detected as
-# in the future and an fsck will be done.  Setting this to 0 results
-# in an epoch of January 1, 1970 which is detected as an invalid time
-# and the fsck will be skipped.
-PACKAGECONFIG:append:mender-systemd = " set-time-epoch"
-SOURCE_DATE_EPOCH:mender-systemd = "0"
-- 
2.34.1

